test/integration/tasks.test.js

```diff
@@ -299,12 +299,63 @@ describe("Tasks", function () {
         .set("cookie", `${cookieName}=${jwt}`)
         .send({
           title: "new-title",
+          dependsOn: ["dependency1", "dependency2"],
         })
         .end((err, res) => {
           if (err) {
             return done(err);
           }
           expect(res).to.have.status(204);
+          return done();
+        });
+    });
+    it("Should update dependency", async function () {
+      const taskData = {
+        title: "Test task",
+        type: "feature",
+        endsOn: 1234,
+        startedOn: 4567,
+        status: "COMPLETED",
+        dependsOn: ["taskId2", "taskId3"],
+        percentCompleted: 100,
+        participants: [],
+        assignee: appOwner.username,
+        completionAward: { [DINERO]: 3, [NEELAM]: 300 },
+        lossRate: { [DINERO]: 1 },
+        isNoteworthy: true,
+      };
```

Comment:
it would be better if this can come from a fixture

Reply:
done